crates/provider-blobstore-fs/src/lib.rs

```diff
@@ -563,6 +563,12 @@ impl bindings::exports::wrpc::blobstore::blobstore::Handler<Option<Context>> for
         Ok(async {
             propagate_trace_for_ctx!(cx);
             let path = self.get_object(cx, id).await?;
+            if let Some(parent) = path.parent() {
+                info!(parent = ?parent.display(), "creating directory");
```

Comment:
```suggestion
                info!(parent = ?parent.display(), "parent directory did not exist, attempting to create");
```

nit: wording that might help point to what is going on. Thoughts? I was going to suggest debug level, but since this should only happen once I'm, good with info